civis/KYC/kyc_permits_to_esri_v2.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+#--Pulling Last 6 Months of Commercial & Apartment Data
+
+
+import os
+pwd=os.getcwd()
+import sys
+#!{sys.executable} -m pip install sodapy
+from sodapy import Socrata
+import pandas as pd
+import numpy as np
+from arcgis.gis import GIS
+from shapely.geometry import Point
+import geopandas as gpd
+from arcgis.features import FeatureLayerCollection
+import intake_civis
+import datetime
+
+lahub_user = os.environ["LAHUB_ACC_USERNAME"]
+lahub_pass = os.environ["LAHUB_ACC_PASSWORD"]
+socrata_token = 'LJ60SFL7ZqoC4IWosLhEmJV2a'
```

Comment:
May be worth also putting this in an environment variable, if it should be considered sensitive.